imu:
  topic: "/livox/imu"
  hz: 200

lidar:
  topic: "/livox/lidar"
  N_SCANS: 6
  hz: 10

camera:
  topic: "/gige_cam/image_raw/compressed"
  img_width: 1920
  img_height: 1200
  hz: 15
  camera_intrinsic:
      [1206.7897947013153, 0.0, 955.3385368757114,
      0.0,  1206.799990244726, 593.511553506258,
      0.0, 0.0, 1.0 ] 
  camera_dist_coeffs: [-0.09430411687002506, 0.08241827862093772, -6.458957311806168e-05, -0.00042875603412696604]  #k1, k2, p1, p2, k3
  equalize: 1 # 如果光太亮或太暗则为1，进行直方图均衡化


slmod_vio:
  # camera_2_inertial_R:
  #        [0.00718201,-0.999964,-0.00462555,
  #        -0.0039344,,0.00459738,-0.999982,
  #        0.999966,0.00720008,-0.00390124]

  camera_2_inertial_R:
         [0.00718201,-0.0039344,0.999966,
         -0.999964,0.00459738,0.00720008,
         -0.00462555,-0.999982,-0.00390124]
  camera_2_inertial_t: [0,0,0] 

object:
  topic: "/yolov5_ros/objects"